aws/dms/tasks.yml

```diff
@@ -24,6 +24,9 @@ cron-pii:
 - dashboard.circuit_playground_discount_codes
 - dashboard.projects
 - dashboard.foorm_%
+- dashboard.learning_goal_ai_evaluations
+- dashboard.learning_goal_teacher_evaluations
```

Comment:
Is there a reason to include the free-form text `feedback` column in this export?

Reply:
yes. we are planning to analyze these data.